pkg/feature/templates/servicemesh/authorino/auth-smm.tmpl

```diff
@@ -0,0 +1,10 @@
+apiVersion: maistra.io/v1
+kind: ServiceMeshMember
+metadata:
+  name: default
+  namespace: {{ .AppNamespace }}
```

Comment:
@bartoszmajsak So, with the NPs issues, I wonder if you foresee any issues because of this namespace being enrolled in the mesh?

Reply:
Reverted to installing Authorino to its own namespace.